src/components/TextArea/index.js

```diff
@@ -8,27 +8,21 @@ import StyledTextArea from './styles/StyledTextArea'
  * A textarea.
  *
  */
-class TextArea extends PureComponent {
-  onChangeValue = e => {
-    const { onChange } = this.props
+const TextArea = ({ className, id, name, onChange, placeholder, ...rest }) => {
+  const onChangeValue = e => {
     onChange && onChange(e.target.value)
   }
 
-  render() {
-    // eslint-disable-next-line no-unused-vars
-    const { className, id, name, onChange, placeholder, ...rest } = this.props
-
-    return (
-      <StyledTextArea
-        className={className}
-        id={id}
-        placeholder={placeholder}
-        name={name}
-        onChange={this.onChangeValue}
-        {...rest}
-      />
-    )
-  }
+  return (
+    <StyledTextArea
+      className={className}
+      id={id}
+      placeholder={placeholder}
+      name={name}
+      onChange={onChangeValue}
+      {...rest}
```

Comment:
Using `React.memo()` would be justified here too in my opinion, because of the `{...rest}` prop passed